subsys/nfc/lib/platform.c

```diff
@@ -190,3 +323,57 @@ void nfc_platform_event_handler(nrfx_nfct_evt_t const *event)
 		break;
 	}
 }
+
+
+#ifdef CONFIG_NFC_THREAD_CALLBACK
+void nfc_platform_cb_request(void *ctx,
+			     size_t ctx_len,
+			     const uint8_t *data,
+			     size_t data_len,
+			     bool copy_data)
+{
+	uint32_t item_buf_tx[MAX_ITEM_SIZE32];
+	size_t item_len;
+	uint8_t item_len32;
+	int err;
+	uint8_t *buf = (uint8_t *)item_buf_tx;
+
+	item_len = ctx_len;
+
+	memcpy(buf, ctx, ctx_len);
```

Comment:
Do you think it's possible to skip the copying to the intermediate stack buffer (`item_buf_tx`)? The ring buf performs the copying again to its internal buffer.

Reply:
Currently trying to find a solution.

Reply:
This can be a good idea, I would really like to avoid this intermediate copy. Just please add a comment in the code to justify the double put operation.